bootstrap/bootstrap.py

```diff
@@ -108,6 +76,40 @@ def validateargs(self):
             raise Exception("Project name should be 3 to 15 chars long")
```

Comment:
Now that we're replacing more things, I think our valid set of chars might be reduced. \w and _ are probably it.